kde-misc/kdialog/kdialog-22.12.2.recipe

```diff
@@ -0,0 +1,118 @@
+SUMMARY="Display dialog boxes from shell scripts"
+DESCRIPTION="kdialog allows you to display dialog boxes from shell scripts.
+The syntax is very much inspired from the 'dialog' command
+(which shows text mode dialogs)."
```

Comment:
The text should have line continuations(backslashes) at 100 characters to wrap the lines.